src/clients/python/__init__.py

```diff
@@ -535,7 +535,8 @@ class ResultFormat:
         RAW = 1,
         CLASS = 2
 
-    def __init__(self, url, protocol, model_name, model_version=None, verbose=False, correlation_id=0):
+    def __init__(self, url, protocol, model_name, model_version=None,
+                 streaming=False, verbose=False, correlation_id=0):
```

Comment:
Need to put streaming as last parameter or you will break compatibility with code that wasn't using named parameters.